app/matplotlib.ipynb

```diff
@@ -200,7 +221,9 @@
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
     "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, autopct=\"%10.0f%%\", startangle=0)\n",
+    "\n",
+    "plt.title(\"Pie plot\")"
```

Comment:
You should add `plt.show()` at the end of this cell to ensure the pie plot is displayed in the notebook.